src/io/flutter/utils/UrlUtils.java

```diff
@@ -0,0 +1,22 @@
+package io.flutter.utils;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+public class UrlUtils {
+    public static String addUrlTags(String input) {
+        StringBuilder builder = new StringBuilder();
+        for (String token : input.split(" ")) {
+            if (builder.length() > 0) {
+                builder.append(" ");
+            }
+            try {
+                URL url = new URL(token);
```

Comment:
in general, having code that throws a bunch of exceptions in its normal operation is annoying. Is there a method you can call to check whether a token is a valid URL without trying and catching? If there is not and this is used sparingly, fine to leave as is as IntelliJ already throws tons of exceptions.

Reply:
We could use an apache validator instead? It would require importing a new library to our plugin. Otherwise I think this is okay since it's only triggered when the prefix is found.